dnf.spec

```diff
@@ -37,20 +37,27 @@
 %bcond_without python2
 %endif
 
-# configurable name for the compat yum package
-%global yum_subpackage_name %{name}-yum
-
-# provide nextgen-yum4 on rhel <= 7 to avoid conflict with existing yum
+# YUM compat subpackage configuration
+#
+# level=full    -> deploy all compat symlinks (conflicts with yum < 4)
+# level=minimal -> deploy a subset of compat symlinks only
+#                  (no conflict with yum >= 3.4.3-505)*
+# level=preview -> minimal level with altered paths (no conflict with yum < 4)
+# *release 505 renamed /usr/bin/yum to /usr/bin/yum-deprecated
+%global yum_compat_level full
+%global yum_subpackage_name yum
+%if 0%{?fedora}
+    %global yum_compat_level minimal
+    %if 0%{?fedora} < 31
+        # Avoid name conflict with yum < 4
+        %global yum_subpackage_name %{name}-yum
+    %endif
+%endif
 %if 0%{?rhel} && 0%{?rhel} <= 7
+    %global yum_compat_level preview
     %global yum_subpackage_name nextgen-yum4
```

Comment:
This is _still_ a stupid name for the package in EL7...

Reply:
I agree.  But there are reasons why it was chosen (mostly to prevent users from installing it by accident) and since it's released there already, I'm afraid we can't change it now (or it's not worth the trouble).